source/nodejs/adaptivecards-designer-app/package.json

```diff
@@ -20,7 +20,7 @@
 	"dependencies": {
 		"adaptivecards": "^3.0.0-beta.1",
 		"adaptivecards-designer": "^2.4.0-alpha.0",
-		"adaptivecards-templating": "^2.3.0-alpha.0",
```

Comment:
revert this change. we never publish 2.3.0-alpha. you can check it from https://www.npmjs.com/package/adaptivecards-templating

Reply:
thanks for pointing this out! just reverted the change